src/components/menu/menu.stories.tsx

```diff
@@ -0,0 +1,542 @@
+import React, { useState } from "react";
+import { ComponentStory } from "@storybook/react";
+
+import {
+  Menu,
+  MenuItem,
+  MenuDivider,
+  MenuSegmentTitle,
+  ScrollableBlock,
+  MenuFullscreen,
+} from ".";
+import Box from "../box";
+import useMediaQuery from "../../hooks/useMediaQuery";
+import Search, { SearchEvent } from "../search";
+import Typography from "../typography";
+import { MenuType } from "./menu.context";
+
+const menuTypes: MenuType[] = ["white", "light", "dark", "black"];
+
+const renderMenu = (children: React.ReactNode) => (
+  <Box mb={150}>
+    {menuTypes.map((menuType) => (
+      <Box key={menuType}>
+        <Typography variant="h4" textTransform="capitalize" my={2}>
+          {menuType}
+        </Typography>
+        <Menu menuType={menuType}>{children}</Menu>
+      </Box>
+    ))}
+  </Box>
+);
```

Comment:
What do you think about moving the JSX from this function into each of the stories. While this would duplicate repeating code it may be less confusing for consumers reading the story source code in the docs who won't know what the `renderMenu` function is.

Reply:
yeah good point I hadn't considered that the code blocks would prevent a DRY approach here